src/utils/network_info.ts

```diff
@@ -787,6 +809,7 @@ export const cosmosNetworkKeys = [
   "islamic_coin",
   "jackal",
   "kava",
+  "kava-testnet",
```

Comment:
These properties are not filtered out in prod, so here they should be added dynamically (something like `.concat(ENABLE_TESTNETS ? ["kava-testnet", "stargaze-testnet"] : []) `)

Reply:
4e0cfcb35391a0b091f6b34e3b33fc09f27c3bdc